yarn-project/noir-contracts/src/contracts/noir-aztec3/src/state_vars/storage_field.nr

```diff
@@ -1,17 +1,31 @@
 use crate::notes::value_note::Note;
 use crate::types::point::Point;
 
-#[builtin(storageRead)]
-fn storage_read(
+#[oracle(storageRead)]
+fn storage_read_oracle(
     _storage_slot: Field,
 ) -> Field {}
 
-#[builtin(storageWrite)]
-fn storage_write(
+unconstrained fn storage_read(
+    storage_slot: Field,
+) -> Field {
+    storage_read_oracle(storage_slot)
+}
```

Comment:
Do we need this function, if `storage_read_oracle` is being called in `StorageField.read` which is already `unconstrained`? Same for write.

Reply:
True! removing those